Makefile

```diff
@@ -1,68 +1,134 @@
+.DEFAULT_TARGET: help
 
-# include .env file and export its env vars
-# (-include to ignore error if it does not exist)
+# Import the .env files and export their values (ignore any error if missing)
 -include .env
+-include .env.dev
 
-# linux: allow shell scripts to be executed
-allow-scripts:; chmod +x ./coverage.sh
+# Set the RPC URL's for each target
+test-fork-testnet: export RPC_URL = $(TESTNET_RPC_URL)
```

Comment:
Again, I can see what you're doing here having all the instantiation of env vars in the same place - was jumping around a bit to follow the make dependencies but now I get it - maybe we move them next to each other?

Reply:
Done 👍 
I have left only 3 at the top, because they might need to be changed and I think it's good that they remain easy to spot.